msteams-platform/concepts/design/enable-app-customization.md

```diff
@@ -29,27 +32,25 @@ You can't test this feature during development. App customization isn't supporte
 
 Provide guidelines for customers (specifically Teams admins) who want to customize your app. For more information, see [customize apps in Teams](/MicrosoftTeams/customize-apps).
 
-## Hide Teams app until admin approves
+## Hide Teams app from users
 
 To enhance Teams app experience, you can hide an app from users by default until admin allows to unhide the app. For example, Contoso Electronics has created a help desk app for Teams. To enable appropriate functioning of the app, Contoso Electronics’ wants the customers to first set up specific properties of the app. The app is hidden by default and is available to users only after the admin allows it.
 
 > [!NOTE]
-> Teams store has evolved:
-> 
-> Previously, the LOB apps were updated by selecting the ellipses on the tile. With the updated Teams store experience, you can now update the LOB apps by logging in to the [Teams Admin Centre](https://admin.teams.microsoft.com).
+> The Teams store has evolved. Earlier, updating the LOB apps involved selecting the ellipses on the tile. You may now update the LOB apps by logging into the [Teams Admin Centre](https://admin.teams.microsoft.com) with the updated Teams store experience.
```

Comment:
Why is this note written so badly? :-( Please re-read as a novice user and see if anything in it makes sense.

* Store is always evolving. No need to mention in docs.
* We typically don't mention what happened earlier. 
  * The only time we mention the previous/earlier state of the product is in changelog or release notes or what's new or rarely in help article if the feature is very new.
* No clue to the reader about the tile on which ellipses were selected. Which tile? Which UI on what surface?
  * Redundant info anyway, so don't mention.
* Stop writing now/currently/earlier/etc. in docs. Check my channel post about this.
* It is `Teams admin center` and not `Teams Admin Center`. Notice the casing in the product.
* How come updated store experience is helping admins update apps in TAC?

Reply:
Hi @ashishguptaiitb, Thanks for the feedback. I noted all the points and will take care of them in the future doc updates. I read through the doc context again, I feel the **Note** doesn't seem to give any important guidance or limitations. Considering the context, can I remove this note?

Reply:
Sure @v-ajiths , please remove the note.